crates/release_plz/src/main.rs

```diff
@@ -44,13 +44,18 @@ async fn run(args: CliArgs) -> anyhow::Result<()> {
             let cargo_metadata = cmd_args.update.cargo_metadata()?;
             let config = cmd_args.update.config()?;
             let update_request = cmd_args.update.update_request(&config, cargo_metadata)?;
-            let request = get_release_pr_req(&config, update_request)?;
-            let release_pr = release_plz_core::release_pr(&request).await?;
+            let requests = get_release_pr_reqs(&config, update_request)?;
+
+            let mut prs = vec![];
+            for request in &requests {
+                let release_pr = release_plz_core::release_pr(request).await?;
+
+                if let Some(pr) = release_pr {
+                    prs.push(pr);
+                }
+            }
+
```

Comment:
`prs` is only used at line 60. Can you calculate `prs` iat line 59 as it was before?
Maybe you can have a vector of `release_prs` outside of the `if let Some` 👍